features/xpath.yml

```
@@ -0,0 +1,25 @@
+name: XPath
+description: "XML Path Language (`XPath`) uses a syntax akin to file paths to provide a flexible way of pointing to or selecting different parts of an XML document. It can also be used to test addressed nodes within a document to determine whether they match a pattern or not."
```

So the platform feature here is the API used to execute XPath queries, not so much the language itself (XPath in the browsers is relatively antiquated with respect to XPath as it is specified today). I think this might be a good way to describe it (kinda like we do for [JSON](https://github.com/web-platform-dx/web-features/blob/45bbdebd61a32ec8242bdbe6eb26b14dbb9704a3/features/json.yml)):

```suggestion
description: "The `document.evaluate()` method selects elements in an HTML or XML document based on an expression of XPath, a domain specific language for querying XML documents. Also known as XML Path Language."
```